eureka/src/main/java/com/linecorp/armeria/client/eureka/EurekaEndpointGroup.java

```diff
@@ -85,6 +89,13 @@
     private static final String SVIPS = "/svips/";
     private static final String INSTANCES = "/instances/";
 
+    /**
+     * The {@link AttributeKey} that store eureka {@link InstanceInfo}.
+     * Use {@link Endpoint#attr(AttributeKey)} to retrieve the value.
+     */
+    private static final AttributeKey<InstanceInfo> INSTANCE_INFO = AttributeKey.valueOf(
+            EurekaEndpointGroup.class, "instanceInfo");
```

Comment:
```suggestion
    static final AttributeKey<InstanceInfo> INSTANCE_INFO = AttributeKey.valueOf(
            EurekaEndpointGroup.class, "INSTANCE_INFO");
```
- We conventionally use the field name.
- Let's remove `private` and use the field in the test code.